src/state/slices/backdrop_slice.ts

```diff
@@ -80,15 +91,23 @@ export const addBackdropDerivedStateSubscribers = (store: SubscribableStore<Back
   );
 };
 
-export const serializeBackdropSlice = (state: BackdropSlice): SerializedStoreData => {
+export const serializeBackdropSlice = (state: Partial<BackdropSliceSerializableState>): SerializedStoreData => {
   const ret: SerializedStoreData = {};
-  ret[UrlParam.SHOW_BACKDROP] = encodeBoolean(state.backdropVisible);
-  ret[UrlParam.BACKDROP_BRIGHTNESS] = encodeNumber(state.backdropBrightness);
-  ret[UrlParam.BACKDROP_SATURATION] = encodeNumber(state.backdropSaturation);
-  ret[UrlParam.OBJECT_OPACITY] = encodeNumber(state.objectOpacity);
+  ret[UrlParam.SHOW_BACKDROP] = encodeMaybeBoolean(state.backdropVisible);
+  ret[UrlParam.BACKDROP_BRIGHTNESS] = encodeMaybeNumber(state.backdropBrightness);
+  ret[UrlParam.BACKDROP_SATURATION] = encodeMaybeNumber(state.backdropSaturation);
+  ret[UrlParam.OBJECT_OPACITY] = encodeMaybeNumber(state.objectOpacity);
   return ret;
 };
 
+/** Selects state values that serialization depends on. */
+export const backdropSliceSerializationDependencies = (slice: BackdropSlice): BackdropSliceSerializableState => ({
+  backdropVisible: slice.backdropVisible,
+  backdropBrightness: slice.backdropBrightness,
+  backdropSaturation: slice.backdropSaturation,
+  objectOpacity: slice.objectOpacity,
+});
+
```

Comment:
Eventually when URL sync is set up, I'll want the URL to update ONLY when serializable values change. 
I set up a selector in each slice file to specify which values URL syncing should be listening for changes to.